client/a8c-for-agencies/sections/purchases/licenses/license-preview/index.tsx

```diff
@@ -286,9 +293,15 @@ export default function LicensePreview( {
 							isChildLicense={ isChildLicense }
 						/>
 					) : (
-						<Button onClick={ open } className="license-preview__toggle" borderless>
-							<Gridicon icon={ isOpen ? 'chevron-up' : 'chevron-down' } />
-						</Button>
+						/*
+						 * For all pressable licenses, only the owner has access to the action,
+						 * so only show the actions if you are the owner or if this is not a pressable license.
+						 */
+						( isOwner || ! isPressableLicense ) && (
+							<Button onClick={ open } className="license-preview__toggle" borderless>
+								<Gridicon icon={ isOpen ? 'chevron-up' : 'chevron-down' } />
+							</Button>
+						)
```

Comment:
We have already disabled the link above to manage it on the Pressable side. Should we disable the preview information here too?

As per the issue information, we only have to disable the link:

> We should add some messaging to users who aren't the agency owner, telling them that they need to ask the agency owner for access

> However, as a quick win, perhaps we could change the "Manage on Pressable" link to a label that says "Managed by Agency owner" and remove that link to Pressable for now.

Reply:
@cleacos The preview doesn't offer any other actions for team members since they are also unable to revoke a license. I included logic to simply hide it because preview information is unnecessary, and no license ID is disabled.

Reply:
I'll add a description of this.